### ISSUE TITLE
Any plans for video driven motion?

### ISSUE THREAD

**Issue by gunnarmarino** [yerfor/GeneFacePlusPlus#53]
This is a great project! I'm very impressed by the output quality. I'm just wondering, is there any plans for adding the ability to drive the model with video instead of audio? If not, I'm going to try to implement it myself, any tips to point me in the right direction would be much appreciated.

Also, I noticed this:
https://github.com/yerfor/GeneFacePlusPlus/blob/98fa99265b6ca82bed97f3bd2b300ae80da4140f/inference/genefacepp_infer.py#L523
 (translates to ~"Currently, only the pose from the source video is supported, including from the beginning and a specified interval. Temporarily does not support in-the-wild pose.").
Could you share any further details about this?

**Comment by yerfor** [comment]
Yes, you can implement yourself. Just replace the `audio2secc` step in the inference script that predicts the `exp` code from audio, instead, write a method like `video2secc`, in which your can directly use our `fit_3dmm_landmark` script to obtain `exp` from a video. Then run the `secc2video` step to obtain the final video!

As for the head pose, you can see in the inference script that the head pose is obtained by indexing the training dataset. So we are using the GT head poses. You can also implement yourself to support external video reference to provide head poses.  But it may leads to worse quality though.
